with-monorepo/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "scripts": {
+    "postinstall": "./postinstall.js"
```

Comment:
i would recommend this instead: https://github.com/expo/expo/blob/master/package.json#L10